test/hook.afterPackageCreateDeploymentArtifacts.test.js

```diff
@@ -74,4 +74,149 @@ describe('Serverless warmup plugin after:deploy:deploy hook', () => {
 
     expect(fs.remove).not.toHaveBeenCalled()
   })
+
+  it('Should have default global package property', async () => {
+    const mockProvider = { request: jest.fn(() => Promise.resolve()) }
+    const serverless = getServerlessConfig({
+      getProvider () { return mockProvider },
+      service: {
+        custom: {
+          warmup: {
+            enabled: true
+          }
+        },
+        functions: { someFunc1: { name: 'someFunc1' }, someFunc2: { name: 'someFunc2' } }
+      }
+    })
+    const plugin = new WarmUp(serverless, {})
+
+    await plugin.hooks['after:package:initialize']()
+
+    expect(plugin.serverless.service.functions.warmUpPlugin.package)
```

Comment:
This only test the package.
Just for consistency with the rest of the tests, to make sure that this config doesn't affect other configs and because using the `getExpectedFunctionConfig` helper is basically free, you should compare the whole config.

```js
    expect(plugin.serverless.service.functions.warmUpPlugin)
      .toEqual(getExpectedFunctionConfig({
        package: {
          individually: true,
          exclude: ['**'],
          include: ['_warmup/**']
        }
      }))
```

or since that's the default
```js
    expect(plugin.serverless.service.functions.warmUpPlugin)
      .toEqual(getExpectedFunctionConfig())
```

Same applies to all other tests.

Reply:
ah yes, good point. fixed